test/test_modified_scripts.py

```diff
@@ -52,11 +46,8 @@ def get_modified_scripts():
         local_script, local_version = item.lower().split(",")
         # check for new scripts or a change in versions for present scripts
         # repo script versions compared with upstream.
-        if local_script not in upstream_versions.keys():
+        if LooseVersion(local_version) != upstream_versions[local_script]:
```

Comment:
I think this should be greater than since if we have an outdated script locally, it probably wasn't modified and we probably don't care to test it. Maybe @henrykironde has more to say on this though

Reply:
I agree with you @zhangcandrew.